src/Cosmos.Kernel.System.Graphics/Fonts/BIOSFont.cs

```diff
@@ -0,0 +1,109 @@
+            fixed (char* ptr = VGA816)
+            {
+                return ptr;
+            }
+        }
+    }
+
+    public static uint* Framebuffer;
+    private static bool _Initialized;
+
+    private static uint Pitch;
+    private static byte* FontData;
+
+    public override int CharWidth => 8;
+
+    public override int CharHeight => 16;
+
+    private static void EnsureInitialized()
+    {
+        if (!_Initialized)
+        {
+            byte* fontData = Base64.Decode(Default.GetUnmanagedFontData(), (uint)Default.Size);
```

Comment:
We _should_ be able to use a `ReadOnlySpan` to store the bytes directly, without the need for Base64-decoding. For example:

https://github.com/ascpixi/smolsharp/blob/54a4aeb6e8f6cdd64d049862dee611301082e71b/src/SmolSharp.Ocean/Shaders/OceanShader.cs#L20-L56